controllers/update_status.go

```diff
@@ -287,9 +289,8 @@ func (s UpdateStatus) Reconcile(r *FoundationDBClusterReconciler, context ctx.Co
 		status.FailingPods = nil
 	}
 
-	if len(status.StorageServersPerDisk) == 0 {
```

Comment:
This case can never happen since we add the default value: https://github.com/FoundationDB/fdb-kubernetes-operator/pull/442/files#diff-291a96adb6e62f693eebc44f9b612d5eff1fcf23f5479a5151340afb91dbd118R51